scripts/oci_integration_tests.sh

```diff
@@ -86,6 +86,13 @@ check_environment() {
   fi
 }
 
+if [[ ! -e $RUNTIME ]]; then
+  if ! which $RUNTIME ; then
+    echo "$RUNTIME not found"
+    exit 1
+  fi
+fi
+
```

Comment:
Shouldn't we set `$RUNTIME=$1` here like it is done in the `rust_integration_tests.sh` ?

Reply:
Is this?
https://github.com/utam0k/youki/blob/0758953fd754452cf385c3bbb52fa813fed28420/scripts/oci_integration_tests.sh#L5

Reply:
oh yes, I missed that :sweat_smile: